### DISCUSSION TITLE
I'm having problems running the sample code for BC C# -

### DISCUSSION THREAD

**Discussion by lawrenceh-ssl** [bcgit/bc-csharp#587] [Q&A]
it can't seem to find the symbol definitions, etc. I am running Win11 (current release), and .NET SDK 8.0.404 and SDK 9.0.101 - I'm using Visual Studio Pro 17.12.3. I installed BouncyCastle.cryptography using NuGet package manager and it appears to work. I've tried running the my-dotnet.cspro code on the documentation page, but it keeps saying "The target "ManifestResourceNames" does not exist in the project. I can't find any step by step guide to getting BC programs to run. Help?

**Comment by peterdettman** [comment]
To what example code are you referring?
